ipmininet/ipnet.py

```diff
@@ -549,6 +550,64 @@ def ping6Pair(self):
            return: ploss packet loss percentage"""
         return self.pingPair(use_v4=False)
 
+   
+    def runFailurePlan(self,failure_plan):
```

Comment:
Please use type hinting (https://docs.python.org/3/library/typing.html) for your methods

For instance, here, I think that it is
```suggestion
    def runFailurePlan(self, failure_plan: List[Tuple[str, str]]) -> List[IPIntf]:
```